### ISSUE TITLE
Could not read source map for file:///home/owner/cp/project/node_modules/typescript/lib/typescript.js: ENOENT: no such file or directory, open '/home/owner/cp/project/node_modules/typescript/lib/typescript.js.map'

### ISSUE THREAD

**Issue by SeanDez** [microsoft/vscode#103006]
VSCode Version: $ code
--version
1.47.1
485c41f9460bdb830c4da12c102daff275415b53
x64
OS Version: Lubuntu 20.04
Steps to Reproduce:

1. clone; https://github.com/SeanDez/foreignSentenceRepeater
2. run npm i to pull deps
3. run `tsc` in terminal (the preLaunchTask is having its own issue, so is not being used)
4. Go to the Run view with CTRL SHIFT D, select `test:once` and run this configuration. You should then see the debugger errors.

The debugger does eventually show the expected test failure so this isn't a breaking error, but it creates a lot of noise and maybe hidden issues:

Does this issue occur when all extensions are disabled?: Yes

```
/home/owner/.config/nvm/versions/node/v14.2.0/bin/node /home/owner/cp/foreignSentenceRepeater/node_modules/jest/bin/jest.js
Debugger listening on ws://127.0.0.1:33231/1a3261a3-78ba-4d3a-a721-fef9e8c17102
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
Debugger attached.
Debugger attached.
Debugger attached.
Debugger attached.
Debugger attached.
Debugger attached.
Debugger attached.
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/typescript/lib/typescript.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/typescript/lib/typescript.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/typescript/lib/typescript.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/typescript/lib/typescript.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/typescript/lib/typescript.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/typescript/lib/typescript.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/typescript/lib/typescript.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/typescript/lib/typescript.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/typescript/lib/typescript.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/typescript/lib/typescript.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@grpc/proto-loader/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@grpc/proto-loader/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@grpc/proto-loader/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@grpc/proto-loader/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/googleauth.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/googleauth.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/googleauth.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/googleauth.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/crypto.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/crypto.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/browser/crypto.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/browser/crypto.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/node/crypto.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/node/crypto.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/messages.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/messages.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/transporters.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/transporters.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/options.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/options.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/computeclient.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/computeclient.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/oauth2client.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/oauth2client.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/authclient.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/authclient.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/crypto.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/crypto.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/browser/crypto.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/browser/crypto.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/node/crypto.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/crypto/node/crypto.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/messages.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/messages.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/transporters.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/transporters.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/options.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/options.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/computeclient.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/computeclient.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/oauth2client.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/oauth2client.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/authclient.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/authclient.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/loginticket.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/loginticket.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/idtokenclient.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/idtokenclient.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/envDetect.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/envDetect.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/jwtclient.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/jwtclient.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/node_modules/gtoken/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/node_modules/gtoken/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/jwtaccess.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/jwtaccess.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/loginticket.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/loginticket.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/idtokenclient.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/idtokenclient.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/envDetect.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/envDetect.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/jwtclient.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/jwtclient.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/node_modules/gtoken/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/node_modules/gtoken/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/jwtaccess.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/jwtaccess.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/refreshclient.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/refreshclient.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/iam.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/iam.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/refreshclient.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/refreshclient.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/iam.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/google-auth-library/build/src/auth/iam.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v2/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v2/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v2/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v2/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/operation.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/operation.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/operation.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/operation.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/service-object.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/service-object.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/service-object.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/service-object.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/promisify/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/promisify/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/util.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/util.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/projectify/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/projectify/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/service.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/service.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/promisify/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/promisify/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/util.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/util.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/projectify/build/src/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/projectify/build/src/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/service.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/common/build/src/service.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3beta1/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3beta1/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3beta1/translation_service_client.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3beta1/translation_service_client.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3beta1/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3beta1/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3beta1/translation_service_client.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3beta1/translation_service_client.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3/translation_service_client.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3/translation_service_client.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3/index.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3/index.js.map'
Could not read source map for file:///home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3/translation_service_client.js: ENOENT: no such file or directory, open '/home/owner/cp/foreignSentenceRepeater/node_modules/@google-cloud/translate/build/src/v3/translation_service_client.js.map'
Debugger attached.
A worker process has failed to exit gracefully and has been force exited. This is likely caused by tests leaking due to improper teardown. Try running with --runInBand --detectOpenHandles to find leaks.
Debugger listening on ws://127.0.0.1:44815/2a15d125-f622-4d6d-9d5b-660e16f6aa20
For help, see: https://nodejs.org/en/docs/inspector
Debugger listening on ws://127.0.0.1:35767/49f5594b-843d-455a-9977-10b333e7b37d
For help, see: https://nodejs.org/en/docs/inspector
Debugger listening on ws://127.0.0.1:43095/fbbc9ef2-3282-49f3-9565-47f06f075149
For help, see: https://nodejs.org/en/docs/inspector
Debugger listening on ws://127.0.0.1:34773/1bea47f4-ecc8-438b-b2ff-707d074df6ab
For help, see: https://nodejs.org/en/docs/inspector
Debugger listening on ws://127.0.0.1:45727/e3a643f4-13a2-4911-a252-92cea881f9f3
For help, see: https://nodejs.org/en/docs/inspector
Debugger listening on ws://127.0.0.1:45817/77b28c46-ecfe-40d0-8db0-ff3490e4ba84
For help, see: https://nodejs.org/en/docs/inspector
Debugger listening on ws://127.0.0.1:33585/cc9a370e-e86d-4c18-9d4b-ca5c16da544a
For help, see: https://nodejs.org/en/docs/inspector
PASS src/classes/sentenceBuilder/__tests__/Sentence.test.ts (36.553 s)
PASS src/classes/setupWizard/__tests__/SetupRole.test.ts (38.428 s)
PASS src/classes/setupWizard/__tests__/SetupWizard.test.ts (38.888 s)
PASS src/classes/sentenceBuilder/__tests__/BuildOrchestrator.test.ts (45.331 s)
FAIL src/classes/sentenceBuilder/__tests__/AudioMaker.test.ts (45.709 s)
  ● textTranslate (gcloud integration)

    Error: A target language is required to perform a translation.

      206 |       return translations[0];
      207 |     } catch (error) {
    > 208 |       throw new Error(error);
          |             ^
      209 |     }
      210 |   }
      211 | 

      at AudioMakerAllPublic.<anonymous> (src/classes/sentenceBuilder/AudioMaker.ts:208:13)
      at step (src/classes/sentenceBuilder/AudioMaker.ts:33:23)
      at Object.throw (src/classes/sentenceBuilder/AudioMaker.ts:14:53)
      at rejected (src/classes/sentenceBuilder/AudioMaker.ts:6:65)

Debugger listening on ws://127.0.0.1:43095/fbbc9ef2-3282-49f3-9565-47f06f075149
For help, see: https://nodejs.org/en/docs/inspector
Waiting for the debugger to disconnect...
Debugger listening on ws://127.0.0.1:34773/1bea47f4-ecc8-438b-b2ff-707d074df6ab
For help, see: https://nodejs.org/en/docs/inspector
Debugger listening on ws://127.0.0.1:33585/cc9a370e-e86d-4c18-9d4b-ca5c16da544a
For help, see: https://nodejs.org/en/docs/inspector
Waiting for the debugger to disconnect...
Debugger listening on ws://127.0.0.1:35767/49f5594b-843d-455a-9977-10b333e7b37d
For help, see: https://nodejs.org/en/docs/inspector
Debugger listening on ws://127.0.0.1:44815/2a15d125-f622-4d6d-9d5b-660e16f6aa20
For help, see: https://nodejs.org/en/docs/inspector
undefined:0



illegal access
FATAL ERROR: 
node::inspector::Agent::ToggleAsyncHook Cannot toggle Inspector's AsyncHook, please report this.
Waiting for the debugger to disconnect...
Waiting for the debugger to disconnect...
 1: 0xa295e0 node::Abort() [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
 2: 0x9782df node::FatalError(char const*, char const*) [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
 3: 0x9782e8  [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
 4: 0xb1d3c8  [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
 5: 0x11f7544 v8_inspector::V8Debugger::setAsyncCallStackDepth(v8_inspector::V8DebuggerAgentImpl*, int) [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
 6: 0x11fc841 v8_inspector::V8DebuggerAgentImpl::disable() [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
 7: 0x121887f v8_inspector::V8InspectorSessionImpl::~V8InspectorSessionImpl() [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
 8: 0x1218b21 v8_inspector::V8InspectorSessionImpl::~V8InspectorSessionImpl() [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
 9: 0xb19804  [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
10: 0xb19ca1  [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
11: 0xb37b5a  [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
12: 0xb39651 node::inspector::MainThreadInterface::RemoveObject(int) [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
13: 0xb3794e  [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
14: 0xb37d6a  [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
15: 0x9ceaf8 node::Environment::RunAndClearInterrupts() [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
16: 0x9cebbc node::Environment::RunAndClearNativeImmediates(bool) [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
17: 0x9cf2f9 node::Environment::CleanupHandles() [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
18: 0x9cf657 node::Environment::RunCleanup() [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
19: 0x996f87 node::FreeEnvironment(node::Environment*) [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
20: 0xa6a1ff node::NodeMainInstance::Run() [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
21: 0x9f9651 node::Start(int, char**) [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
22: 0x7fad665ec0b3 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
23: 0x992f0c  [/home/owner/.config/nvm/versions/node/v14.2.0/bin/node]
Debugger listening on ws://127.0.0.1:34437/e2fc129a-d6ad-4a1e-959d-b1feeed12247
For help, see: https://nodejs.org/en/docs/inspector
Waiting for the debugger to disconnect...
A worker process has failed to exit gracefully and has been force exited. This is likely caused by tests leaking due to improper teardown. Try running with --runInBand --detectOpenHandles to find leaks.
Test Suites: 1 failed, 4 passed, 5 total
Tests:       1 failed, 1 skipped, 6 passed, 8 total
Snapshots:   0 total
Time:        79.18 s
Ran all test suites.
Debugger listening on ws://127.0.0.1:33231/1a3261a3-78ba-4d3a-a721-fef9e8c17102
For help, see: https://nodejs.org/en/docs/inspector
Process exited with code 1
```

**Comment by SeanDez** [comment]
```
// launch config
{
         "type": "node",
         "request": "launch",
         "name": "test:once",
         "skipFiles": [
            "<node_internals>/**"
         ],
         "program": "${workspaceFolder}/node_modules/jest/bin/jest.js",
         // "preLaunchTask": "tsc",
         "args": [""],
         "outFiles": [
            "${workspaceFolder}/compiled/**/*.js"
         ],
         "sourceMaps": true,
         "smartStep": true,
         "internalConsoleOptions": "openOnSessionStart"
      }

// tsconfig.json
{
   "compilerOptions": {
       "target" : "ES5",
       "module": "commonjs",
       "noImplicitAny": true,
       "removeComments": true,
       "sourceMap": true
       , "outDir": "compiled/"
       , "esModuleInterop": true
       , "strict": true
       , "typeRoots": ["./src/types", "node_modules/@types"]
   },
   "include": ["src/**/*"]
   , "exclude": ["node_modules", "compiled", "__tests__", "types"]
}
```

**Comment by connor4312** [comment]
This is fixed in VS Code 1.47.2. 

/duplicate https://github.com/microsoft/vscode/issues/102042
